### ISSUE TITLE
Jigasi should encrypt SIP credentials

### ISSUE THREAD

**Issue by edouardduliege** [jitsi/jigasi#266]
It appears Jigasi stores SIP credentials in plain text. these should be encrypted for obvious security reasons.

**Comment by saghul** [comment]
Not that obvious. The crentials are stored on a file on the filesystem. If an attacker gets accesss to your server you have bigger problems, don't you/

**Comment by edouardduliege** [comment]
>If an attacker gets accesss to your server you have bigger problems, don't you?
correct, I would tend to not add another problem but not having encrypted credentials though.

If you think this is not something to worry, feel free to close the issue.

**Comment by x676f64** [comment]
@saghul consider that users are often terrible about picking unique passwords across services and systems.  Password reuse is a real thing and once credentials are compromised, they may be leveraged for attacks against other systems within the environment.

**Comment by eddie-planetnet** [comment]
My feeling is if you arent going to store encrypted, why require them to be stored base64? It seems like a nuisance.
